docker/local/buid_image.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+TEMP_FOLDER=/tmp/build_wb
+PATH_TO_WB_REPO=$(pwd)
+HTTP_PROXY=${http_proxy}
+HTTPS_PROXY=${https_proxy}
+IMAGE_NAME=workbench
+IMAGE_TAG=local
+OPENVINO_VERSION="automation/Jenkins/openvino_version.yml"
+
+while (( "$#" )); do
+  case "$1" in
+    --wb-path)
+      PATH_TO_WB_REPO=$2
+      shift 2
+      ;;
+    --base-image)
+      BASE_IMAGE=$2
+      shift 2
+      ;;
+    --wheels-path)
+      WHEELS_PATH=$2
+      shift 2
+      ;;
+    --bundles-path)
+      BUNDLES_PATH=$2
+      shift 2
+      ;;
+    --http-proxy)
+      HTTP_PROXY=$2
+      shift 2
+      ;;
+    --https-proxy)
+      HTTPS_PROXY=$2
+      shift 2
+      ;;
+    *)
+      echo Unsupport argument $1
+      exit 1
+      ;;
+  esac
+done
+
+set -e
+
+FOLDER=$(basename $PATH_TO_WB_REPO)
```

Comment:
The folder with repository can have any name, I suggest a warning instead of `exit`.